docs/advanced_topics/third_party_tutorials.rst

```diff
@@ -5,7 +5,7 @@ Third-party tutorials
 
     The following list is a collection of tutorials and development notes from third-party developers.
     Some of the older links may not apply to the latest Wagtail versions.
-
+* `Image upload in wagtail forms<https://posts-by.lb.ee/image-uploads-in-wagtail-forms-3121c9b35d27>`_ - Tutorial on adding image uploads to wagtail forms(3 September 2017) 
```

Comment:
Hey, thanks for linking to this by the way - appreciate it.

I do have plans to re-write some of these tutorials for the new Wagtail but not sure when I will get to it.

Anyway, just a few typos here, there should be a space just before the opening bracket of the link, also there should be a space before the opening bracket of the date (to align with the others that have been added.

If you can fix those minor things up I will get this merged in asap.